webcompat/static/js/lib/bugform.js

```diff
@@ -227,6 +232,29 @@ function BugForm() {
     }
   };
 
+  this.clearStepsToReproducePlaceholder = function() {
+    var stepsToReproduceStrAr = this.stepsToReproduceField
+      .val()
+      .split("4. Nothing happened.");
+
+    if (stepsToReproduceStrAr.length > 1 && stepsToReproduceStrAr[1] === "\n") {
```

Comment:
The more I look at this approach, the more I get confused... it's not obvious to me what the linebreak means in the 2nd position of the array unless I open up a console and mess around. 

`stepsToReproduceStrAr.length > 1 && stepsToReproduceStrAr[1].length > 1` is even more opaque.

I wonder if it would be way simple if we tried a different approach... always clear out the placeholder on focus, and only add the `details` param stuff to the end `onFormSubmit().

Or can you think of a way to do this that's going to be easier to understand when we re-visit the code in the future?